app/Http/Review/ReviewController.php

```diff
@@ -32,7 +32,7 @@ public function __construct(ReviewRepository $reviews)
      */
     public function index()
     {
-        $reviews = $this->reviews->getAll();
+        $reviews = $this->reviews->getAllReviews();
```

Comment:
It seems like you have renamed the `getAll()` method to `getAllReviews()`. Please ensure that the `getAllReviews()` method is properly defined in the `ReviewRepository` class and it is doing the same job as `getAll()`. If not, this could potentially break the code.